main.go

```diff
@@ -22,6 +22,14 @@ const (
 	CloudWatchNamespace = "Skpr/CloudFront"
 )
 
+var (
+	// FiveMinutesAgo is a variable storing time. It will be used to make a
+	// time comparison between the time an invalidation was created and
+	// five minutes ago, which is the fixed input time which this lambda
+	// is intended to execute.
+	FiveMinutesAgo = time.Now().Add(time.Minute * -5)
```

Comment:
I think this should go at the top of the "Execute" func and not as a global.

Reply:
This is to avoid a bug where the handler is triggered more than once by Lambda

Reply:
So this was done so that testing could continue unhindered.
Given it's just returning a bool and it's called only once I can omit that now.